core/statistics/machine/cpuStatistics.go

```diff
@@ -17,17 +15,19 @@ type CpuStatistics struct {
 // ComputeStatistics computes the current cpu usage. It should be called on a go routine as it is a blocking
 // call for a bounded time (1 second)
 func (cs *CpuStatistics) ComputeStatistics() {
-	cpuUsagePercent, err := cpu.Percent(durationSecond, false)
+	currentProcess, err := getCurrentProcess()
 	if err != nil {
 		cs.setZeroStatsAndWait()
 		return
 	}
-	if len(cpuUsagePercent) == 0 {
+
+	cpuUsagePercent, err := currentProcess.Percent(500 * time.Millisecond)
```

Comment:
magic number

Reply:
Magic number was removed. Now Cpu percent is get for 1 second interval.